pipeline/preprocessing/formatting/FCD_Formatter.py

```diff
@@ -0,0 +1,195 @@
+from pandas import DataFrame, Series  # type: ignore
+from typing import Any
+from io import open
+import json
+from pipeline.preprocessing.compute_features.type_alias import Trip
+
+SegmentToCoordinateDict = dict[int, list[Trip]]
+SegmentToCoordinatesList = list[tuple[int, list]]
+
+
+class FCD_Formatter:
+    @classmethod
+    def from_json_file(self, file_path: str) -> DataFrame:
+        """Loads a file and calls from_json_string on it
+
+        Args:
+            fcd_formatter (FCD_Formatter): The class instance
+            file_path (str): Path to a json file
+
+        Returns:
+            DataFrame: A dataframe with osm_id and segments
+        """
+        with open(file_path, "r") as data_from_fcd:
+            return self.from_json_string(data_from_fcd.read())
+
+    @staticmethod
+    def from_json_string(json_string: str) -> DataFrame:
+        """Converts a FCD data geojson string into a Pandas Dataframe that correlates osm_id to their trips
+
+        Args:
+            json_string (str): FCD data geojson formatted string
+
+        Returns:
+            DataFrame: A dataframe with osm_id and segments
+        """
+        data = json.loads(json_string)  # Load from string
+        data = _remove_fcd_request_wrapper(
+            data
+        )  # Remove the outer layer of json object
+        data = _create_segment_to_coordinate_df(
+            data
+        )  # Convert from geojson format to osm_id -> trips
+
+        return data
+
+
+def _remove_fcd_request_wrapper(wrapdata: Any) -> DataFrame:
```

Comment:
Private functions skal have double underscore. én underscore indikerer protected.

Reply:
Samme for de resterende funktioner

Reply:
Så kan vi ik test